src/js/components/Chart/index.d.ts

```diff
@@ -19,7 +19,7 @@ export interface ChartProps {
   color?:
     | ColorType
     | { color: ColorType; value: number | number[] }[]
-    // deprecated 
+    // deprecated
```

Comment:
Can we try to avoid the distraction of file changes that are not relevant for this PR?

Reply:
I believe it is a formatting change implemented by the linter on commit. I did not touch the file.

Reply:
It looks like these changes came in on this commit which was a merge of master: https://github.com/grommet/grommet/pull/4426/commits/8873ff0c64f3ab9836effa8b71540f6762d6b114